VoteModel.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+Kyle Cloud
+January 17, 2020
+* Updated January 19, 2020 - add model functionality
+
+Prerequisite: Train 7 models.
+
+The purpose of this class is to
+have 7 trained models vote on how
+to classify an image of fruit.
+"""
+
+import numpy as np
+from statistics import mode
+
+class VoteModel():
+    
+    def __init__(self, models): # take in 7 models
+        self.__models = models
+        
+    # added on January 19, 2020
+    def add_model(self, model):
+        self.__models.append(model)
+        
+    def classify(self, image, num2fruit):
+        votes = []
+        for model in self.__models:
+            prob_dist = model.predict(image) # softmax outputs a list
+            vote = np.argmax(prob_dist) # index of max probability
```

Comment:
This function returns the mode of the indices of the maximum values in each probability distribution.
We might also consider taking the maximum of the sums of the probability values for each index.